rest-api/src/main/resources/definitions/demographic_values_validation_config.yml

```diff
@@ -1,8 +1,8 @@
 description: >
-  Configuration stored in app config which configures the validation of demographic values.
-  To enable validation of demographics, this object should be stored in an app config element under
-  the key "bridge-validation-demographics-values-{categoryName}", where "categoryName" is the name
-  of the category which this validation configuration applies to.
+  Configuration which configures the validation of app-level and study-level demographic values.
```

Comment:
@conundrumgirl do the changes to this comment block lose any information that's still relevant and applicable, i.e. under what name the object should be stored?